Sources/Stripe/Models/EphemeralKey/EphemeralKey.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  EphemeralKey.swift
+//  Stripe
+//
+//  Created by Andrew Edwards on 10/17/17.
+//
+
+import Foundation
+import Vapor
+
+public final class EphemeralKey: StripeModelProtocol {
+    
+    public private(set) var id: String?
+    public private(set) var object: String?
+    public private(set) var associatedObjects: Node?
+    public private(set) var created: Date?
+    public private(set) var expires: Date?
+    public private(set) var isLive: Bool?
+    public private(set) var secret: String?
+    
+    public init(node: Node) throws {
+        self.id = try node.get("id")
+        self.object = try node.get("object")
+        self.associatedObjects = try node.get("associated_objects")
+        self.created = try node.get("created")
+        self.expires = try node.get("expires")
+        self.isLive = try node.get("livemode")
+        self.secret = try node.get("secret")
+    }
+    
+    public func makeNode(in context: Context?) throws -> Node {
+        let object: [String: Any?] = [
+            "id": self.id,
+            "object": self.object,
+            "associated_objects": self.associatedObjects,
+            "created": self.created,
+            "expires": self.expires,
+            "livemode": self.isLive,
+            "secret": self.isLive
```

Comment:
@Andrewangeta I think this is a bug. It should be `"secret": self.secret`. The iOS integration is crashing for me because of this.

Reply:
Yep! Nice catch, will change right away.

Reply:
👍

Reply:
Should be good now. Circle CI is running integration tests but it’s safe to keep using.

Reply:
Cool. Is the new version going out as a part of CI?

Reply:
Nah no new version needed. Just run vapor update to get the changes. It’ll still be a part of 1.0.0 you may have to delete your package.pins/resolved first though.

Reply:
Great. Thanks again for implementing this. 🙇

Reply:
@Andrewangeta I just updated and I still have the old code. I cleaned the package things before that.

Reply:
Try again. Delete the build folder entirely, and make sure you have Xcode closed while you run update.

Reply:
Why not tag it as 1.0.1?

Reply:
1.0.0 is a pretty round number lol, plus it Hans’t been pulled by in the few hours it’s been available really.